kolibri/deployment/default/settings/base.py

```diff
@@ -441,3 +442,28 @@
 
 # whether Kolibri is running within tests
 TESTING = False
+
+
+# Content Security Policy header settings
+# https://django-csp.readthedocs.io/en/latest/configuration.html
+CSP_DEFAULT_SRC = ("'self'", "data:", "blob:") + tuple(
```

Comment:
> Is anything too loosely defined?

Should we not set a more restrictive `script-src` policy? It seems data and blob URI for scripts is still overly broad and I'm not sure we even use those?